src/app/item-page/orcid-page/orcid-sync-settings/orcid-sync-settings.component.ts

```diff
@@ -156,7 +156,8 @@ export class OrcidSyncSettingsComponent implements OnInit {
         }
       }),
     ).subscribe((remoteData: RemoteData<ResearcherProfile>) => {
-      if (remoteData.isSuccess) {
+      // hasSucceeded is true if the response is success or successStale
+      if (remoteData.hasSucceeded) {
```

Comment:
This change seems unrelated to this PR?

Reply:
I think this should be removed or explained? It doesn't seem to have anything to do with Importing SAF files.